mogp_emulator/tests/test_GaussianProcess.py

```diff
@@ -433,6 +433,123 @@ def test_GaussianProcess_partial_devs():
     partials_actual = gp.partial_devs(new_theta)
     assert_allclose(partials_actual, partials_expected, rtol = 1.e-5, atol = 1.e-8)
 
+def test_GaussianProcess_hessian():
```

Comment:
Perhaps split this long test into three?

Reply:
Done